t/02-pod.t

```diff
@@ -1,8 +1,9 @@
 use Test::Most;
 use Test::Warnings qw(:no_end_test :report_warnings);
+use Feature::Compat::Try;
 # no OpenQA::Test::TimeLimit for this trivial test
 
-eval 'use Test::Pod';
-plan skip_all => "Test::Pod required for testing POD" if $@;
+try { use Test::Pod }
```

Comment:
That will execute `use Test::Pod` at compile time.
```suggestion
try { require Test::Pod; import Test::Pod }
```
Alternatively leave the `import` out and call `Test::Pod::all_pod_files_ok()` below

Reply:
Another alternative:
```
eval 'use Test::Pod; 1' or plan skip_all => "Test::Pod required for testing POD";
```